src/app/features/ledger/hooks/use-ledger-navigate.ts

```diff
@@ -31,9 +31,22 @@ export function useLedgerNavigate() {
       },
 
       toConnectAndSignBitcoinTransactionStep(psbt: Uint8Array) {
+        // DO NOT MERGE BEFORE FIXED. Running into very weird routing issue,
+        // recording shared on Slack, where I can't actually get the signing
+        // routes nested under the inscription `choose-fee` route, even thoguh
+        // it has the routes defined and an <Outlet /> present.
+        if (location.pathname.includes('inscription')) {
+          return navigate('../' + RouteUrls.ConnectLedger, {
```

Comment:
For some reason, I can get to the route when it isn't off ChooseFee screen.

Reply:
```
export const sendOrdinalRoutes = (
  <Route path={RouteUrls.SendOrdinalInscription} element={<SendInscriptionContainer />}>
    {ledgerBitcoinTxSigningRoutes} // these work

    <Route index element={<SendInscriptionForm />} />
    <Route path={RouteUrls.SendOrdinalInscriptionChooseFee} element={<SendInscriptionChooseFee />}>
      {ledgerBitcoinTxSigningRoutes} // these dont
    </Route>
    <Route path={RouteUrls.SendOrdinalInscriptionReview} element={<SendInscriptionReview />} />

    <Route path={RouteUrls.SendOrdinalInscriptionSent} element={<SendInscriptionSummary />} />
    <Route path={RouteUrls.SendOrdinalInscriptionError} element={<BroadcastError />} />
  </Route>
);
```

Reply:
Maybe due to [this](https://github.com/leather-wallet/extension/pull/4316#discussion_r1384789781) ? 

It could be something weird to do with nesting ` {ledgerBitcoinTxSigningRoutes}` underneath `<Route path={RouteUrls.SendOrdinalInscriptionChooseFee}`

Reply:
Did you see `ModalBackgroundWrapper`? 

Check in `home.tsx` / `fund.tsx`. If you wrap `SendInscriptionChooseFee` with that it might work if it's not related to the `loading`